openhub/management/commands/import_affiliated_committers_data.py

```diff
@@ -0,0 +1,74 @@
+import json
+import requests
+import logging
+import xmltodict
+
+from community.git import get_owner
+from openhub.oh_token import OH_TOKEN
+from django.core.management.base import BaseCommand
+from openhub.models import MostCommit, MostRecentCommit, AffiliatedCommitter
+
+logging.basicConfig(level=logging.DEBUG)
+
+n = 100
+for i in range(1, 2):
+    import_url = 'https://www.openhub.net/orgs/'+get_owner()+'/affiliated_committers.xml?api_key='+OH_TOKEN+'&page='+str(i)
+    resp = requests.get(import_url)
+    jsonString = json.dumps(xmltodict.parse(resp.content), indent=4)
+    json_object = json.loads(jsonString)
+    try:
+        data = json_object['response']['result']['affiliated_committers']['affiliator']
+    except Exception as ex:
+        logging.debug(ex)
+        break
+    class Command(BaseCommand):
+        @staticmethod
```

Comment:
If it doesnt use `self`, it shouldnt be inside the class.  `staticmethod` usually means you've designed it wrong.